pennylane/templates/subroutines/approx_time_evolution.py

```diff
@@ -24,12 +24,6 @@ class ApproxTimeEvolution(Operation):
     r"""Applies the Trotterized time-evolution operator for an arbitrary Hamiltonian, expressed in terms
     of Pauli gates.
 
-    .. note::
-
-        This operation is a special case of :class:`~pennylane.TrotterProduct` and will be deprecated in
-        future releases, please use :class:`~pennylane.TrotterProduct` as the preferred operation for this
-        functionality.
-
```

Comment:
Can we still have a "See also", like here: https://xanaduai-pennylane--4735.com.readthedocs.build/en/4735/code/api/pennylane.ControlledSequence.html

Reply:
Added 👍🏼